### ISSUE TITLE
Bottom toolbar container background color

### ISSUE THREAD

**Issue by talenelat-elin** [eustachi0/RemNoteThemes#1]
Amazing theme, as usual!

Small bug i noticed:
![image](https://user-images.githubusercontent.com/71895835/120898172-54a4db80-c647-11eb-8d3f-a3a109fd4bca.png)
The area around the corners of the toolbar is colored white.

The following snippet should fix it. I didn't know where exactly to put it though, so I'm submitting an issue instead of a PR.

```css
#hierarchy-editor-toolbar-container
{background-color: var(--background-color);}
```

**Comment by eustachi0** [comment]
Hey Thank you very much. I appreciate it
Is that happening on Windows?

Cheers!!
Eustachio

**Comment by talenelat-elin** [comment]
Yep
Windows 10
Desktop App version 1.3.13

The problem is much more visible when we make the window very narrow
![image](https://user-images.githubusercontent.com/71895835/120920192-46080400-c6db-11eb-8848-37a849a241fd.png)

**Comment by talenelat-elin** [comment]
The snippet doesn't fix it all when the window is very narrow. 
This thin white line remains:
![image](https://user-images.githubusercontent.com/71895835/120920318-ca5a8700-c6db-11eb-9b0e-bce00b737d87.png)

**Comment by eustachi0** [comment]
Thank you for this. I'm debugging the code, I will find a fix for this.
Thanks for reporting it.
Cheers,
Eustachio

**Comment by talenelat-elin** [comment]
![image](https://user-images.githubusercontent.com/71895835/120923062-1d3b3b00-c6ea-11eb-9c0b-95a118d79583.png)

The padding or the margins of the highlights is a bit wonky.

**Comment by eustachi0** [comment]
This has been fixed in the version 1.1 released 10 Jun 2021
